pages/tutorials/Quick-Start-Guide.md

```diff
@@ -0,0 +1,95 @@
+> This guide is intended for the **Bukkit** version of Terra. Guides will be written for other platforms once they
```

Comment:
/Spigot/Paper/Tuinity/etc but paper is recommended etc etc.

maybe one day people will figure out how forks work.